packages/vendure-order-client/test/vendure-order-client.spec.ts

```diff
@@ -59,6 +86,66 @@ describe('Vendure order client', () => {
     expect(server.app.getHttpServer).toBeDefined;
   });
 
+  it('Create a promotion', async () => {
```

Comment:
Minor spelling improvement
```suggestion
  it('Creates a promotion', async () => {
```